BUILDING.md

```diff
@@ -35,7 +35,7 @@ file a new issue.
     * [Building Node.js](#building-nodejs-1)
   * [Android/Android-based devices (e.g. Firefox OS)](#androidandroid-based-devices-eg-firefox-os)
 * [`Intl` (ECMA-402) support](#intl-ecma-402-support)
-  * [Default: `small-icu` (English only) support](#default-small-icu-english-only-support)
+  * [Trimmed: `small-icu` (English only) support](#trimmed-small-icu-english-only-support)
```

Comment:
It seems the TOC also needs updating now: this and the next sections neeeds to be swapped, Windows subsection for "Trimmed" needs to be added.

Reply:
And maybe we also need "Unix/macOS" subsection for "Trimmed" to be consistent with the surrounding sections.